src/main/java/com/fundation/logic/model/MetadataExtractor.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.model;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * Implements the MetadataExtractor class
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class MetadataExtractor {
+
+    /**
+     * This method returns metadata frame rate.
+     */
+    public String getFrameRate(File path) {
```

Comment:
I think methods could be static, this way there will be no need to instantiate an object MetadataExtractor and just use the methods.